axelor-base/src/main/java/com/axelor/apps/base/web/LanguageServiceImpl.java

```diff
@@ -0,0 +1,90 @@
+import com.axelor.meta.db.MetaSelectItem;
+import com.axelor.meta.db.repo.MetaSelectItemRepository;
+import com.axelor.meta.db.repo.MetaSelectRepository;
+import com.google.inject.Inject;
+import com.google.inject.persist.Transactional;
+import java.util.List;
+
+public class LanguageServiceImpl implements LanguageService {
+  protected MetaSelectRepository metaSelectRepository;
+  protected MetaSelectItemRepository metaSelectItemRepository;
+  protected LanguageRepository languageRepository;
+
+  @Transactional
+  @Override
+  public void convertToLowercase(Language language) {
+    Language languageObj = Beans.get(LanguageRepository.class).find(language.getId());
+    languageObj.setCode(languageObj.getCode().toLowerCase());
+    languageRepository.save(languageObj);
+  }
+
+  @Inject
+  public LanguageServiceImpl(
+      MetaSelectRepository metaSelectRepository,
+      MetaSelectItemRepository metaSelectItemRepository,
+      LanguageRepository languageRepository) {
+    this.metaSelectRepository = metaSelectRepository;
+    this.metaSelectItemRepository = metaSelectItemRepository;
+    this.languageRepository = languageRepository;
+  }
+
+  @Transactional
+  @Override
+  public void saveExistingLanguageObjectToMetaSelect(
+      Language language, String oldName, String oldCode) {
+    Query<MetaSelect> metaSelectQuery =
+        metaSelectRepository.all().filter("self.name = :selectName");
+    metaSelectQuery.bind("selectName", "select.language");
+    List<MetaSelect> metaSelectList = metaSelectQuery.fetch();
+    MetaSelect metaSelect = metaSelectList.get(0);
+
+    Query<MetaSelectItem> metaSelectItemQuery =
+        metaSelectItemRepository.all().filter("self.title = :oldName and self.value = :oldCode");
+    metaSelectItemQuery.bind("oldName", oldName).bind("oldCode", oldCode);
+    List<MetaSelectItem> metaSelectItemList = metaSelectItemQuery.fetch();
+    MetaSelectItem metaSelectItem = metaSelectItemList.get(0);
+    metaSelectItem.setTitle(language.getName());
+    metaSelectItem.setValue(language.getCode());
+    metaSelectItem.setSelect(metaSelect);
+    metaSelect.addItem(metaSelectItem);
+    metaSelectItemRepository.save(metaSelectItem);
+    metaSelectRepository.save(metaSelect);
+  }
+
+  @Transactional
+  @Override
+  public void saveNewLanguageObjectToMetaSelect(Language language) {
+    Query<MetaSelect> metaSelectQuery =
+        metaSelectRepository.all().filter("self.name = :selectName");
+    metaSelectQuery.bind("selectName", "select.language");
+    List<MetaSelect> metaSelectList = metaSelectQuery.fetch();
+    MetaSelect metaSelect = metaSelectList.get(0);
+
+    MetaSelectItem metaSelectItem = new MetaSelectItem();
+    metaSelectItem.setTitle(language.getName());
+    metaSelectItem.setValue(language.getCode());
+    metaSelectItem.setSelect(metaSelect);
+    metaSelect.addItem(metaSelectItem);
+    metaSelectItemRepository.save(metaSelectItem);
+    metaSelectRepository.save(metaSelect);
+  }
```

Comment:
This method is very similar to the previous one, just in this one you don't find the MetaSelectItem. 
It's better to just have one method and as parameter you can add a boolean findItem, if the boolean is true then you will find the MetaSelectItem else you create a new one